pkg/tests/ossm/smcp_annotation_test.go

```diff
@@ -86,6 +86,9 @@ func GetPodAnnotations(t TestHelper, podLocator oc.PodLocatorFunc) map[string]st
 		if err != nil {
 			t.Fatalf("Error parsing pod annotations json: %v", err)
 		}
+		if len(annotations) == 0 {
+			t.Fatal("Pod annotations are empty, retrying...")
```

Comment:
Don't say "retrying", because this isn't true on the last attempt. Just say that pod annotations are empty.